libkpod/logs.go

```diff
@@ -0,0 +1,64 @@
+package libkpod
+
+import (
+	"fmt"
+	"path"
+	"strings"
+	"time"
+
+	"github.com/hpcloud/tail"
+)
+
+const (
+	// LogsBaseDir specifies the default directory in which logs are stored
+	logsBaseDir = "/var/log/containers"
```

Comment:
That needs to be removed,  kpod logs should read the log dir from the config.  I'll fix that